src/Administration/Resources/app/administration/src/module/sw-profile/page/sw-profile-index/sw-profile-index.html.twig

```diff
@@ -93,54 +93,30 @@
 
         <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
         {% block sw_profile_index_content_confirm_password_modal %}
-        <sw-modal
+        <sw-verify-user-modal
             v-if="confirmPasswordModal"
-            :title="$tc('sw-profile.index.titleConfirmPasswordModal')"
-            variant="small"
-            @modal-close="onCloseConfirmPasswordModal"
-        >
-            <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
-            {% block sw_profile_index_content_confirm_password_modal_input__confirm_password %}
-            <sw-password-field
-                v-model="confirmPassword"
-                class="sw-settings-user-detail__confirm-password"
-                required
-                name="sw-field--confirm-password"
-                :password-toggle-able="true"
-                :copy-able="false"
-                :label="$tc('sw-profile.index.labelConfirmPassword')"
-                :placeholder="$tc('sw-profile.index.placeholderConfirmPassword')"
-                @keypress.enter="onSubmitConfirmPassword"
-            />
-            {% endblock %}
+            @verified="onSubmitConfirmPassword"
+            @close="onCloseConfirmPasswordModal"
+        />
 
-            <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
-            {% block sw_profile_index_content_confirm_password_modal_footer %}
-            <template #modal-footer>
-                <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
-                {% block sw_profile_index_content_confirm_password_modal_actions_cancel %}
-                <sw-button
-                    size="small"
-                    @click="onCloseConfirmPasswordModal"
-                >
-                    {{ $tc('sw-profile.index.labelButtonCancel') }}
-                </sw-button>
-                {% endblock %}
+        {# @deprecated tag:v6.5.0 - will be removed #}
```

Comment:
Deprecations for 6.5 is not allowed anymore, use 6.6.0 instead, same with other relevant code comments 😉